macros/utils/relative_datediff.sql

```diff
@@ -0,0 +1,13 @@
+{% macro relative_datediff(date1, date2, datepart) %}
+    {{
+        dbt_utils.datediff(
+            date1,
+            dbt_utils.dateadd(
+                'day',
+                '1 - ' ~ dbt_date.date_part('day',date1),
```

Comment:
Nit (2)
Missing space between `'day'` and `date1`, should be
`date_part('day', date1)`